sdk/servicefabric/azure-servicefabric/azure/servicefabric/version.py

```diff
@@ -9,5 +9,5 @@
 # regenerated.
 # --------------------------------------------------------------------------
 
-VERSION = "7.2.0.46"
+VERSION = "8.0.0.0"
```

Comment:
Any idea why this is a 4-part version? Most versions are only 3 parts and there are a number of things that I've noticed that break because of this being 4 parts.

Reply:
As examples see:

https://github.com/Azure/azure-sdk-for-python/pull/18494
https://dev.azure.com/azure-sdk/public/_build/results?buildId=873559&view=logs&j=12f1170f-54f2-53f3-20dd-22fc7dff55f9&t=036ed663-514a-59f3-0c88-23d2cf182147